server/interop_handler.py

```diff
@@ -225,3 +234,21 @@ def odlc_load_queue(self, filename="odlc.json"):
                 obj["auto_submit"] = datetime.fromisoformat(obj["auto_submit"])
                 self.odlc_queued_data[x] = obj
             return {"result": "success"}
+
+    def map_add(self, name: str, image: str):
+        if os.path.isfile(f"assets/map_images/{name}.jpg"):
+            return {"result": "failed: map with this name already exists"}
+        with open(f"assets/map_images/{name}.jpg", "wb") as file:
+            file.write(base64.decodebytes(bytes(image, 'utf-8')))
+        self.map_image = image
+        return {"result": "success"}
+
+    def map_submit(self, name=None):
+        if not name:
+            self.submitted_map = base64.decodebytes(bytes(self.map_image, 'utf-8'))
+            self.client.put_map_image(self.mission_id, self.submitted_map)
+        else:
+            with open(f"assets/map_images/{name}.jpg", "rb") as file:
+                self.submitted_map = file.read()
```

Comment:
What's the point of storing `submitted_map` like the class, instead of locally? It doesn't seem to be used otherwise.

Reply:
My idea was to store every possible map as an image in the filesystem, but to also store the most recent uploaded map within the class itself. This is so that if your frontend wants to display the most recent map or some preview of it, it can easily access the most recent map from the server. Not really necessary, but something I thought would be better.